scripts/protogen.sh

```diff
@@ -24,25 +24,25 @@ rm -rf codegen/protocol-test-codegen-local/build
 ./gradlew -p codegen/protocol-test-codegen-local build
 
 # Delete the generated Package.swift for protocol test packages so they may be seen in Xcode
```

Comment:
`rm -f` is used below so the script does not fail if `Package.swift` is not present.